scripts/create-update.ps1

```diff
@@ -0,0 +1,92 @@
+<#
+Copyright (c) AppDynamics, Inc., and its affiliates 2015
+All rights reserved
+
+Build script for wpt-driver.
+
+REQUIREMENTS:
+- msbuild.exe and python.exe (required by the Chrome compile.cmd) should be in
+  the path
+
+This script has been tested in GitHub's Git Shell.
+Executing it in a regular PowerShell may also work, but you will have to
+setup the path for each dependencies.
+#>
+
+param(
+    [string]$configuration = "Release"
+)
+
+$wptroot = $PSScriptRoot + "\.."
+
+"*** Building wpt-driver with $configuration configuration"
+
+# ZipFiles implementation from:
+# http://stackoverflow.com/a/13302548
+function ZipFiles( $zipFilename, $sourceDir)
+{
+   Add-Type -Assembly System.IO.Compression.FileSystem
+   $compressionLevel = [System.IO.Compression.CompressionLevel]::Optimal
+   [System.IO.Compression.ZipFile]::CreateFromDirectory($sourcedir,
+        $zipfilename, $compressionLevel, $false)
+}
+
+$releasedir = Get-ChildItem $wptroot $configuration
```

Comment:
Like we discussed, this line is problematic if 'Release' directory already doesn't exist. Get-ChildItem only gives a successful result if the directories exist. We need to check if 'Release' exists. If not, we should create one and if it exists, delete and create one.